pose_thumbnails/core.py

```diff
@@ -610,6 +618,41 @@ def _determine_poses(self):
             bpy.context.scene.tool_settings.use_keyframe_insert_auto = auto_insert
 
 
+class POSELIB_OT_sanitize_library_name(bpy.types.Operator):
+    """Make the active pose library name suitable for a character"""
+    bl_idname = 'poselib.library_name_sanitize'
+    bl_label = 'Sanitize library name'
+    bl_options = {'PRESET', 'UNDO'}
+
+    @classmethod
+    def poll(self, context):
+        return (context is not None and
+                context.object and
+                context.object.type == 'ARMATURE' and
+                context.object.pose_library)
+
+    def execute(self, context):
+        addon_prefs = prefs.for_addon(context)
+        if not context.object or not context.object.pose_library:
+            return {'FINISHED'}
```

Comment:
`'FINISHED'` is incorrect here, as the operator didn't do anything. Use `'CANCELLED'` instead.